docs/installing.md

```diff
@@ -0,0 +1,80 @@
+# Running a local Haveno test network
+
+These are the steps needed to setup a local Haveno test network.
+
+## 1. Download dependencies
+
+1. Install git, git-lfs and Java. On ubuntu: `sudo apt install git git-lfs openjdk-11-jdk`
+2. Download the [Monero CLI](https://www.getmonero.org/downloads/#cli). We will need `monerod` and `monero-wallet-rpc`, which are contained in the archive.
+3. Download and install [Bitcoin-Qt](https://bitcoin.org/en/download).
+
+## 2. Build Haveno
+
+1. Download this repository. Run `git clone https://github.com/haveno-dex/haveno.git`
+2. Navigate inside the repository (`cd haveno`) and start the build: `./gradlew build`. It will take some minutes. If you know what you are doing you can modify [WalletConfig.java](core/src/main/java/bisq/core/btc/setup/WalletConfig.java) with custom settings before building.
+
+## 3. Run Monero
+
+1. Copy `monero-wallet-rpc`, which you downloaded with the monero CLI archive earlier (step 1.2) to the root of this repository. Make sure it has the right permissions (if you are in doubt, just run `chmod 777 monero-wallet-rpc`).
+2. Navigate into the folder containing `monerod` and run:
+
+```
+./monerod --stagenet --no-igd --hide-my-port --data-dir stagenet --add-exclusive-node 136.244.105.131:38080 --rpc-login superuser:abctesting123 --rpc-access-control-origins http://localhost:8080 --fixed-difficulty 1
```

Comment:
Reminder to self: people will need a monerod version with the changes to hardforks.cpp included. We could distribute static binaries (at least for linux and mac) and also keep the instructions for the custom build, in case people prefer to compile their own version.

Reply:
Added links to our archives (https://github.com/haveno-dex/monero/releases/tag/testing) and adapted instructions.